### ISSUE TITLE
ST3 + Windows 10 failing to install peroperly

### ISSUE THREAD

**Issue by VJatla** [swdotcom/swdc-sublime#36]
This is a fresh install of ST3 on Windows 10. It fails with the following error
```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 196, in load_module
    m.plugin_loaded()
  File "C:\Users\vj\AppData\Roaming\Sublime Text 3\Installed Packages\CodeTime.sublime-package\Software.py", line 357, in plugin_loaded
    initializeUser()
  File "C:\Users\vj\AppData\Roaming\Sublime Text 3\Installed Packages\CodeTime.sublime-package\Software.py", line 390, in initializeUser
    result = createAnonymousUser()
  File "C:\Users\vj\AppData\Roaming\Sublime Text 3\Installed Packages\CodeTime.sublime-package\lib/SoftwareUtil.py", line 333, in createAnonymousUser
    plugin_uuid = getPluginUuid()
  File "C:\Users\vj\AppData\Roaming\Sublime Text 3\Installed Packages\CodeTime.sublime-package\lib/CommonUtil.py", line 89, in getPluginUuid
    jsonObj["plugin_uuid"] = value
NameError: global name 'value' is not defined
```

**Comment by xavluiz** [comment]
Thank you for the issue update. We'll look at this right away.

Regards.

**Comment by xavluiz** [comment]
We apologize for the issue. The issue has been fixed and will be available in v2.3.12.

Regards.

**Comment by VJatla** [comment]
I am closing this issue. I have stopped using subime and started using emacs + wakatime. It works great.
